fix: Update controller-build-scc.yaml

theodor2311 opened the pull request:
Use security.openshift.io/v1 instead of v1 to fix no matches for kind "SecurityContextConstraints" in version "v1" for OpenShift.

theodor2311 committed:
Update controller-build-scc.yaml

Use security.openshift.io/v1 instead of v1 to fix no matches for kind "SecurityContextConstraints" in version "v1" for OpenShift.

theodor2311 commented:
/assign @garethjevans

theodor2311 commented:
/retest

manuelwallrapp commented:
I changed this locally in my config repo jenkins-x-boot-config repo. After I changed it, I still get following error:
error: unable to recognize "/var/folders/6t/vc9qjw1x1yg1kvb_8k04m6cc0000gn/T/helm-template-workdir-230695889/jenkins-x/output/namespaces/jx/env/charts/jxboot-resources/templates/part0-scc-controllerbuild.yaml": no matches for kind "SecurityContextConstraints" in version "v1"'

Will this Yaml from somewhere else applied than the local repo?

theodor2311 commented:
> I changed this locally in my config repo jenkins-x-boot-config repo. After I changed it, I still get following error:
> error: unable to recognize "/var/folders/6t/vc9qjw1x1yg1kvb_8k04m6cc0000gn/T/helm-template-workdir-230695889/jenkins-x/output/namespaces/jx/env/charts/jxboot-resources/templates/part0-scc-controllerbuild.yaml": no matches for kind "SecurityContextConstraints" in version "v1"'
> 
> Will this Yaml from somewhere else applied than the local repo?

It will not work if you only modify this yaml file, I get around by skipping the install-jenkins-x pipeline step and manually applied the charts with modification. I just submitted a PR to the jenkins-x-charts, hope I hit the right file. 
https://github.com/jenkins-x-charts/jxboot-resources/pull/45
